src/utils/extract.js

```diff
@@ -80,5 +80,27 @@ export const extract = {
         }
 
         return '#' + redHex + greenHex + blueHex;
+    },
+    rgbToHSB: (red, green, blue) => {
```

Comment:
Great work here, did you test this?

Reply:
I have personally tested this formula for accuracy against this color converter: http://www.workwithcolor.com/color-converter-01.htm

Reply:
Perfect!